web/src/components/SpeakerCard.vue

```diff
@@ -62,6 +66,10 @@ export default {
       const location = this.prefectures.find((elem) => (elem.id === locationId));
       return location?.fields?.prefecture;
     },
+    languages() {
+      const languageIds = this.speaker.get('languages2');
+      return this.languageList.filter((language) => languageIds.find((id) => (id === language.id)));
```

Comment:
Maybe the condition here could use [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) instead of a `find` function. What do you think?

```javascript
this.languageList.filter((language) => languageIds.includes(language.id));
```